kafka-connect-mysql-v60/Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG KAFKA_CONNECT_VERSION=6.0.2
+
+FROM confluentinc/cp-kafka-connect-base:${KAFKA_CONNECT_VERSION}
+
+ARG KAFKA_CONNECT_VERSION=6.0.2
+ARG MYSQL_CONNECTOR_VERSION=8.0.25
```

Comment:
Is it compatible with mysql 5.6, 5.7?

Reply:
https://dev.mysql.com/doc/refman/8.0/ja/upgrading-from-previous-series.html
I would consider the links incompatible where they pertain to incompatible changes.

However, there is no problem with SQL queries in Kafka Connect. It is working fine with 8 connectors at the moment.